.eslintrc

```diff
@@ -0,0 +1,37 @@
+{
+  "env": {
+    "commonjs": true,
+    "es6": true,
+    "node": true,
+    "mocha": true,
+    "browser": true
+  },
+  "parserOptions": {
+    "sourceType": "module",
+    "allowImportExportEverywhere": true,
+    "ecmaVersion": 2020,
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "ignorePatterns": [
+    "node_modules/**/*",
+    "dist/"
+  ],
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "extends": [
+    "plugin:react/recommended", 
+    "eslint:recommended"
+  ],
+  "rules": {
+    "no-console": "off",
+    "react/prop-types": "off"
+  },
+  "globals": {
+    "window": true
+  }
+}
```

Comment:
I think this file should be ignored in this PR.

Reply:
yes, updated